### ISSUE TITLE
fix: Update links to documentation

### ISSUE THREAD

**Issue by Aviksaikat** [ethersphere/bee-js#894]
In the `bee.js` & `bee-debug.js` file doc links are outdated. 

example 

```sh
// old
https://docs.ethswarm.org/docs/access-the-swarm/upload-and-download

// new
https://docs.ethswarm.org/docs/develop/access-the-swarm/upload-and-download
```

**Comment by Cafe137** [comment]
Thanks for reporting this, fixed in #938

**Comment by bee-worker** [comment]
Release [v7.1.1](https://github.com/ethersphere/bee-js/releases/tag/v7.1.1) addresses this.
